langchain-core/src/tracers/tests/langsmith_interop.test.ts

```diff
@@ -150,21 +150,6 @@ test.each(["true", "false"])(
     });
     expect(thirdCallParams).toMatchObject({
       end_time: expect.any(Number),
-      inputs: {
```

Comment:
Removing `inputs` in these tests is a side-effect of bumping `langsmith` - we were unnecessarily double-sending inputs in the update calls when they were not changing.